README.md

```diff
@@ -41,217 +41,1885 @@ The required view creation scripts are found in:
+(@a_system_run_id, 'HT_000001', 'E3', now(), now()),
+(@a_system_run_id, 'HT_000001', 'E4', now(), now()),
+(@a_system_run_id, 'HT_000001', 'E5', now(), now()),
+(@a_system_run_id, 'HT_000001', 'E6', now(), now()),
+(@a_system_run_id, 'HT_000001', 'E7', now(), now()),
+(@a_system_run_id, 'HT_000001', 'E8', now(), now()),
+(@a_system_run_id, 'HT_000001', 'E9', now(), now()),
+(@a_system_run_id, 'HT_000001', 'E10', now(), now()),
+(@a_system_run_id, 'HT_000001', 'E11', now(), now()),
+(@a_system_run_id, 'HT_000001', 'E12', now(), now()),
+(@a_system_run_id, 'HT_000001', 'F1', now(), now()),
+(@a_system_run_id, 'HT_000001', 'F2', now(), now()),
+(@a_system_run_id, 'HT_000001', 'F3', now(), now()),
+(@a_system_run_id, 'HT_000001', 'F4', now(), now()),
+(@a_system_run_id, 'HT_000001', 'F5', now(), now()),
+(@a_system_run_id, 'HT_000001', 'F6', now(), now()),
+(@a_system_run_id, 'HT_000001', 'F7', now(), now()),
+(@a_system_run_id, 'HT_000001', 'F8', now(), now()),
+(@a_system_run_id, 'HT_000001', 'F9', now(), now()),
+(@a_system_run_id, 'HT_000001', 'F10', now(), now()),
+(@a_system_run_id, 'HT_000001', 'F11', now(), now()),
+(@a_system_run_id, 'HT_000001', 'F12', now(), now()),
+(@a_system_run_id, 'HT_000001', 'G1', now(), now()),
+(@a_system_run_id, 'HT_000001', 'G2', now(), now()),
+(@a_system_run_id, 'HT_000001', 'G3', now(), now()),
+(@a_system_run_id, 'HT_000001', 'G4', now(), now()),
+(@a_system_run_id, 'HT_000001', 'G5', now(), now()),
+(@a_system_run_id, 'HT_000001', 'G6', now(), now()),
+(@a_system_run_id, 'HT_000001', 'G7', now(), now()),
+(@a_system_run_id, 'HT_000001', 'G8', now(), now()),
+(@a_system_run_id, 'HT_000001', 'G9', now(), now()),
+(@a_system_run_id, 'HT_000001', 'G10', now(), now()),
+(@a_system_run_id, 'HT_000001', 'G11', now(), now()),
+(@a_system_run_id, 'HT_000001', 'G12', now(), now()),
+(@a_system_run_id, 'HT_000001', 'H1', now(), now()),
+(@a_system_run_id, 'HT_000001', 'H2', now(), now()),
+(@a_system_run_id, 'HT_000001', 'H3', now(), now()),
+(@a_system_run_id, 'HT_000001', 'H4', now(), now()),
+(@a_system_run_id, 'HT_000001', 'H5', now(), now()),
+(@a_system_run_id, 'HT_000001', 'H6', now(), now()),
+(@a_system_run_id, 'HT_000001', 'H7', now(), now()),
+(@a_system_run_id, 'HT_000001', 'H8', now(), now()),
+(@a_system_run_id, 'HT_000001', 'H9', now(), now()),
+(@a_system_run_id, 'HT_000001', 'H10', now(), now()),
+(@a_system_run_id, 'HT_000001', 'H11', now(), now()),
+(@a_system_run_id, 'HT_000001', 'H12', now(), now())
+;
+```
+
+-- for positive control, check if it is already present
+```sql
+SELECT id FROM `biosero_uat`.`control_plate_wells`
+WHERE barcode = 'Control01'
+AND coordinate = 'A1';
+```
+
+-- if not, insert positive control well into control_plate_wells table
+```sql
+INSERT INTO `biosero_uat`.`control_plate_wells` (
+  automation_system_run_id,
+  barcode,
+  coordinate,
+  control,
+  created_at,
+  updated_at
+)
+VALUES
+(@a_system_run_id, 'Control01', 'A1', 'positive', now(), now())
```

Comment:
Is it worth making the control fiend case insensitive

Reply:
Made it an enum with positive and negative values only